src/components/FeatureLists/FeatureList.tsx

```diff
@@ -3,19 +3,17 @@ import { Flex, View, Heading } from '@aws-amplify/ui-react';
 
 interface FeatureListProps {
   heading: string;
-  level: 1 | 2 | 3 | 4 | 5 | 6;
+  level?: 1 | 2 | 3 | 4 | 5 | 6;
   children: React.ReactNode;
 }
 const FeatureList: React.FC<FeatureListProps> = ({
   children,
   heading,
-  level
+  level = 3
 }) => {
   return (
-    <Flex direction="column" className="category-list">
-      <Heading level={level} className="category-heading-margin">
-        {heading}
-      </Heading>
+    <Flex className="category-list">
```

Comment:
Just some className cleanup. We're already giving it a class name, so moved the direction style prop to css.